docs/descriptors/php_php_cs_fixer.md

```diff
@@ -20,13 +20,6 @@ description: How to use php-cs-fixer (configure, ignore files, ignore errors, he
 - Enable php-cs-fixer by adding `PHP_PHPCSFIXER` in [ENABLE_LINTERS variable](https://megalinter.io/beta/configuration/#activation-and-deactivation)
 - Disable php-cs-fixer by adding `PHP_PHPCSFIXER` in [DISABLE_LINTERS variable](https://megalinter.io/beta/configuration/#activation-and-deactivation)
 
-- Enable **autofixes** by adding `PHP_PHPCSFIXER` in [APPLY_FIXES variable](https://megalinter.io/beta/configuration/#apply-fixes)
```

Comment:
See @llaville ? :)

Reply:
@nvuillam It seems that it is also added when `cli_lint_fix_arg_name` is defined on a linter descriptor file
see https://github.com/oxsecurity/megalinter/blob/v8.1.0/.automation/build.py#L1395-L1399

But it's not used in case of PHP-CS-Fixer

Reply:
@nvuillam see PR #4249